<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClient</class>
 <widget class="QDialog" name="MyClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TetraSnakeClient</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>262</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Server IP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_IP"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_port">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_port"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_recive">
            <property name="text">
             <string>Please, enter socket.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_connect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="line_send"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_send">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Start_Game_Button">
        <property name="text">
         <string>Start Game</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="close_game_button">
      <property name="text">
       <string>Close Game</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
